.travis.yml

```diff
@@ -23,6 +21,7 @@ before_install:
     #   https://github.com/npm/npm/issues/6309
   - nvm install 0.12
   - nvm use 0.12
+  - find . -type l -name "bower_components" -delete -o -name "node_modules" -delete
```

Comment:
This helps by getting rid of the symlinks before NPM and Bower try to do their thing.